sdk/openai/azure-ai-openai/tsp-location.yaml

```diff
@@ -1,5 +1,5 @@
 directory: specification/cognitiveservices/OpenAI.Inference
 additionalDirectories:
     - specification/cognitiveservices/OpenAI.Authoring
-commit: b646a42aa3b7a0ce488d05f1724827ea41d12cf1
+commit: dd2d1e8957ac6654272137e8d5874eacafd80a5f
```

Comment:
@jpalvarezl Please update the commit id to that of main branch, after audio.tsp get merged to main

Reply:
Will do as soon as the [PR](https://github.com/Azure/azure-rest-api-specs/pull/25734) is merged